### PR TITLE
krakend 0.6.0

### PR DESCRIPTION
**Pull request by alombarte**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
==> make build
Building the binary...
# github.com/devopsfaith/krakend-ce
github.com/devopsfaith/krakend-ce/vendor/github.com/tmthrgd/go-hex.encodeASM: relocation target runtime.support_avx not defined
github.com/devopsfaith/krakend-ce/vendor/github.com/tmthrgd/go-memset.memsetAsm: relocation target runtime.support_avx not defined
make: *** [build] Error 2
```

**Comment by alombarte** [comment]
Ooops

The referenced vendor does not work with the latest Go version (1.11), need to use 1.10.3 ...

File: Formula/krakend.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Krakend < Formula
   end
 
   depends_on "dep" => :build
-  depends_on "go" => :build
+  depends_on "go@1.10" => :build
```
Comment (alombarte) [inline Formula/krakend.rb]: This change should be reverted, as the release requires 1.11

Comment (fxcoudert) [reply Formula/krakend.rb]: @alombarte if the new version build with go, we would welcome a pull request

**Pull request closed without merging**